src/lib/ngsiNotify/QueueNotifier.cpp

```diff
@@ -100,8 +101,8 @@ void QueueNotifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const st
     spathList = "";
   }
 
-  ci.outFormat = format;
-  std::string payload = ncr->render(&ci, NotifyContext, "");
+  ci.outFormat = JSON;
```

Comment:
Yeah, I was thinking about that same thing. Actually, mimeType we used ... a long time ago.